packages/langium-cli/src/package.ts

```diff
@@ -83,26 +83,13 @@ export async function loadConfigs(options: GenerateOptions): Promise<LangiumConf
     log('log', options, `Reading config from ${chalk.white.bold(filePath)}`);
     try {
         const obj = await fs.readJson(filePath, { encoding: 'utf-8' });
-        const config: LangiumConfig | LangiumConfig[] = path.basename(filePath) === 'package.json' ? obj.langium : obj;
-        if (Array.isArray(config)) {
-            config.forEach(c => {
-                c[RelativePath] = relativePath;
-            });
-            return setProjectNames(config);
-        } else {
-            config[RelativePath] = relativePath;
-        }
-        return setProjectNames([config]);
+        const config: LangiumConfig = path.basename(filePath) === 'package.json' ? obj.langium : obj;
+        config.projectName = _.camelCase(config.projectName);
+        config.projectName = config.projectName.charAt(0).toUpperCase() + config.projectName.slice(1);
+        config[RelativePath] = relativePath;
+        return config;
```

Comment:
This is a breaking change – shall we defer it to 2.0?

Reply:
I'm not sure whether anyone actually used multiple langium configs in one file (i.e. the ability to generate multiple Langium based projects using a single invocation of the `langium-cli`). It was something I added for testing purposes which we never ended up needing, due to being able to interpret Langium grammars on the fly.

While it was working in theory, the feature was completely undocumented and I'm quite certain that no one used it. I would keep it in this PR for 1.2.0.

Reply:
Ok, but please document its removal in CHANGELOG of langium-cli